app/components/channel_modal.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes, PureComponent} from 'react';
+import {Animated, Dimensions} from 'react-native';
+
+const {View: AnimatedView} = Animated;
+const {width: deviceWidth, height: deviceHeight} = Dimensions.get('window');
```

Comment:
I actually don't have any issues with the syntax, but I know I had to change some of my code cause @hmhealey has an opinion on using this, If he doesn't mind im totally up for using this, actually im 4/5 on using this syntax